### PR TITLE
setup local PostgreSQL

### PR DESCRIPTION
**Pull request by asmaa-ezz**
# Description
setup local PostgreSQL.

add 
 * database Schema #7 
 * fake data 
 * connection database

## Type of change
New features #22
Description : setup local PostgreSQL

### CODE REVIEW

**Review by ali-7 — APPROVED** [review]

**Review by lubnaabd — APPROVED** [review]

**Review by lubnaabd — APPROVED** [review]

File: src/database/dbBulid.sql [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+  CREATE TABLE library (
+    id serial PRIMARY KEY,
+    book_id integer NOT NULL REFERENCES book(id) ON UPDATE CASCADE,
+    bookshelf integer NOT NULL,
+    section integer NOT NULL,
+    copy_id integer NOT NULL
+);
+
+
```
Comment (RamyAlshurafa) [inline src/database/dbBulid.sql]: remove second white line

File: src/database/dbConnection.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+const { Pool } = require('pg');
+const url = require('url');
+require('env2')('./config.env');
+
+let dbUrl = '';
```
Comment (RamyAlshurafa) [inline src/database/dbConnection.js]: Is it needed to declare it here!?
you can declare it inside if block

File: src/database/fakeData.sql [hunk 3]
```diff
@@ -0,0 +1,32 @@
+
```
Comment (RamyAlshurafa) [inline src/database/fakeData.sql]: remove this line

File: src/database/fakeData.sql [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+INSERT INTO admin(user_name, password) VALUES ('user','000');
```
Comment (RamyAlshurafa) [inline src/database/fakeData.sql]: password here must be hashed

File: src/database/dbBuild.js [hunk 5]
```diff
@@ -0,0 +1,14 @@
+const fs = require('fs');
+const path = require('path');
+const dbConnection = require('./dbConnection');
+
+
```
Comment (RamyAlshurafa) [inline src/database/dbBuild.js]: remove this line

**Review by RamyAlshurafa — CHANGES_REQUESTED** [review]
check the comments and fix it
#### use snake_case for files name

File: src/database/dbConnection.js (hunk 2 above)
Comment (ali-7) [reply src/database/dbConnection.js]: it's used in all the file i can't declare it in the if statement scope

Comment (ali-7) [reply src/database/dbConnection.js]: @RamyAlshurafa

Comment (RamyAlshurafa) [reply src/database/dbConnection.js]: that's right

File: src/database/dbBulid.sql [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+CREATE TABLE users (
```
Comment (RamyAlshurafa) [inline src/database/dbBulid.sql]: It's better to make table names as a singular

Comment (asmaa-ezz) [reply src/database/dbBulid.sql]: postgresql can not use "user" name in the table 
I try that but it is error

Comment (asmaa-ezz) [reply src/database/dbBulid.sql]: @RamyAlshurafa

**Review by RamyAlshurafa — APPROVED** [review]

**Pull request merged**